app/services/testing_ground/concurrency/maximizer.rb

```diff
@@ -0,0 +1,13 @@
+class TestingGround
+  class Concurrency
+    module Maximizer
+      def maximize
+        -> (techs) {
+          tech       = techs.first
+          tech.units = techs.sum(&:units)
+          tech
+        }
```

Comment:
Very minor thing: use `-> { }` for single-line blocks, and `lambda do ... end` for multi-line blocks.

https://github.com/bbatsov/ruby-style-guide#lambda-multi-line